qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -0,0 +1,71 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+from qiita_db.study import Study
+from qiita_pet.handlers.api_proxy.util import check_access
+
+
+def data_types_get_req():
+    """Returns data types available in the system
+
+    Returns
+    -------
+    list of str
+        Data types available on the system
+    """
+    return {'status': 'success',
+            'message': '',
+            'data_types': Study.all_data_types()
+            }
+
+
+def study_get_req(study_id, user_id):
+    """Returns information available for the given study
+
+    Parameters
+    ----------
+    study_id : int
+        Study id to get prep template info for
+    user_id : str
+        User requesting the info
+
+    Returns
+    -------
+    dict of {str: object}
```

Comment:
This documentation is actually incorrect. It returns `dict of {'status': str, 'message': str, 'info': dict of {str: object}`

The same applies for the above function.